create_mapping_pan_ancestry_round2_round1.R

```diff
@@ -0,0 +1,59 @@
+library(tidyverse)
+
+prefix <- "/Volumes/rahul/"
```

Comment:
My only high-level comment here is that we should design a system such that everyone can run our code after just cloning. I have some aspects of this in `constants.R` (where it checks for data in a `data/` subfolder and if it doesn't find it or if the size is different from what's on the server, tries to download it - obviously the data has to be able to be public but I think this is).

I am guilty of doing something like this though in the first line of that script in which I assume you have the `constants.R` from the `gnomad_lof` in a certain place, so we should fix that within there.